TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -708,36 +707,39 @@ def makeTreeFromMiniAOD(
     
     # AK8 jet variables - separate instance of jet properties producer
     from TreeMaker.Utils.jetproperties_cfi import jetproperties
+    #JetAK8Tag = cms.InputTag('slimmedJetsAK8')
     process.JetsPropertiesAK8 = jetproperties.clone(
         JetTag       = JetAK8Tag,
         properties = cms.vstring(
             "prunedMass"    ,
             "NsubjettinessTau1"    ,
             "NsubjettinessTau2"    ,
             "NsubjettinessTau3"    ,
-            "bDiscriminatorSubjet1",
-            "bDiscriminatorSubjet2",
+            #"bDiscriminatorSubjet1",
+            #"bDiscriminatorSubjet2",
             "bDiscriminatorCSV"    ,
             "NumBhadrons"          ,
             "NumChadrons"          ,
         )
     )
-    #specify userfloats
-    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSPrunedMass')
-    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('NjettinessAK8:tau1')
-    process.JetsPropertiesAK8.NsubjettinessTau2 = cms.vstring('NjettinessAK8:tau2')
-    process.JetsPropertiesAK8.NsubjettinessTau3 = cms.vstring('NjettinessAK8:tau3')
-    process.JetsPropertiesAK8.bDiscriminatorSubjet1 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
-    process.JetsPropertiesAK8.bDiscriminatorSubjet2 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
-    process.JetsPropertiesAK8.bDiscriminatorCSV = cms.vstring('pfBoostedDoubleSecondaryVertexAK8BJetTags')
+    #specify userfloat
+    #process.JetsPropertiesAK8.prunedMass = cms.vstring('prunedMass')
+    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass')
+    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1')
```

Comment:
Hey Kevin, For the subjettiness and the AK8 jet mass we are currently using the pruned version of these variables (which is what was stored in the previous versions of Treemaker). These variables are still stored in the current datasets as user floats (which I got from your debug flag) :

ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2
ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3
ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass

I conferred with Andrew and the above variables are the ones we want to use.